docs: update RFC-0201/TariScript: Add metadata ComSig to the transaction output (#3070)
tari-project/tari: RFC/src/Glossary.md

verified transactions from the mempool to build up transaction [block]s.

## Metadata Signature
[metadata signature]: #metadata-signature

> The metadata signature is an aggregated Commitment Signature signature, attached to a transaction output, signed with a 
> combination of the homomorphic commitment private values \\( (v\_i \\, , \\, k\_i )\\), with the spending key only known 
> by the receiver, and sender offset private key \\(k\_{Oi}\\), only known by the sender. This stops any form of 
> malleability of the UTXO metadata.

Reviewer: Some minor readability nits 

```suggestion
The metadata signature is an aggregated Commitment Signature ("ComSig") signature, attached to a transaction output and signed with a 
combination of the homomorphic commitment private values \\( (v\_i \\, , \\, k\_i )\\), the spending key known only 
to the receiver, and sender offset private key \\(k\_{Oi}\\) known only to the sender. This prevents malleability of the UTXO metadata.
```

Author: Thanks! Let me fix that.
